tests/unit/cython/types_testhelper.pyx

```diff
@@ -38,7 +38,7 @@ def test_datetype(assert_equal):
         cdef BytesIOReader reader
         cdef Buffer buf
 
-        dt = datetime.datetime.utcfromtimestamp(timestamp)
+        dt = datetime.datetime.fromtimestamp(timestamp, tz=datetime.timezone.utc)
```

Comment:
The original impl (using utcfromtimestamp()) returns a naive datetime and this version doesn't.  But after looking into this  a bit it doesn't actually matter.  This datetime is immediately serialized and then deserialized in order to test the end-to-end process.  All that really matters here are the time values; any timezone info is lost in that process.